NiBAx/plugins/harmonization/harmonization.py

```diff
@@ -345,6 +347,43 @@ def OnAddToDataFrame(self):
         self.datamodel.data.loc[:,RAW_Residuals] = self.MUSE[RAW_Residuals]
         self.datamodel.data_changed.emit()
 
+
+    def OnMUSEharmonizationModelChanged(self):
+        if self.datamodel.IsValidHarmonization():
```

Comment:
This if check seems redundant. When we reach here, we have already done the IsValidHarmonization() check.

Reply:
The MUSE harmonization model may be updated elsewhere. It can be updated but not be valid (i.e. set to None).